site/docs/docs.md

```diff
@@ -47,10 +47,3 @@ In Vega-Lite, a specification can have the following top-level properties.
 | [mark](mark.html)    | String        | The mark type.  One of `"bar"`, `"circle"`, `"square"`, `"tick"`, `"line"`, `"area"`, `"point"`, and `"text"`. |
 | [encoding](encoding.html) | Object   | A key-value mapping between encoding channels and definition of fields. |
 | [config](config.html)   | Object     | Configuration object. |
-
```

Comment:
I don't think it's a good idea to have usage instructions partially in 3 places, but I can put it back (and up) if you think thats better.